polars/Cargo.toml

```diff
@@ -131,7 +131,7 @@ bigidx = ["polars-core/bigidx", "polars-lazy/bigidx", "polars-ops/big_idx"]
 list_to_struct = ["polars-ops/list_to_struct", "polars-lazy/list_to_struct"]
 list_take = ["polars-ops/list_take", "polars-lazy/list_take"]
 describe = ["polars-core/describe"]
-timezones = ["polars-core/timezones", "polars-lazy/timezones"]
+timezones = ["polars-core/timezones", "polars-lazy/timezones", "polars-io/timezones"]
```

Comment:
ah looks like this is what I was missing - thanks for fixing it up!

looks like this needs a rebase?

Reply:
Nope, a squash will do. :+1: